lib/whitehall.rb

```diff
@@ -162,6 +162,10 @@ def self.organisations_in_tagging_beta
       YAML.load_file(Rails.root + "config/organisations_in_tagging_beta.yml")["organisations_in_tagging_beta"]
   end
 
+  def self.tagging_taxonomy_enabled?
```

Comment:
Hum... you are right, we didn't removed it from the base_controller, I had the impression we had done that when we added the method here. I don't think we have a good reason to keep this anyway, will remove it and then I'll open a PR on govuk-puppet to remove the env variable.